Files/Filesystem/INavigationControlItem.cs

```diff
@@ -12,6 +12,7 @@ public enum NavigationControlItemType
     {
         Header,
         Drive,
+        RecycleBin,
```

Comment:
Removing all Windows.UI.* references from the LocationItem requires us to differentiate the RecycleBin item because its glyph uses a different font family.